Plugins/BTCPayServer.RockstarDev.Plugins.XpubExtractor/Resources/js/plugin-vaultbridge.ui.js

```diff
@@ -276,31 +276,42 @@ var vaultui = (function () {
             return true;
         };
 
-        this.askForXPubs = async function () {
+        this.askForXPubs = async function (signatureType, addressType, accountNumber) {
             if (!await self.ensureConnectedToBackend())
                 return false;
+
             self.bridge.socket.send("ask-xpub");
             var json = await self.bridge.waitBackendMessage();
             if (json.hasOwnProperty("error")) {
                 if (await needRetry(json))
-                    return await self.askForXPubs();
+                    return await askForXPubs(signatureType, addressType, accountNumber);
                 return false;
             }
             try {
-                var selectedXPubs = await self.getXpubSettings();
```

Comment:
We shouldn't be changing the signature of `askForXPubs`, rather update `getXpubSettings` to take into consideration the new `customPath` field.